extensions/binding/org.eclipse.smarthome.binding.ntp/src/main/java/org/eclipse/smarthome/binding/ntp/handler/NtpHandler.java

```diff
@@ -227,12 +227,12 @@ public long getTime(String hostname) {
         } catch (UnknownHostException uhe) {
             String msg = getThing().getUID().toString() + " the given hostname '" + hostname
                     + "' of the timeserver is unknown -> returning current sytem time instead.";
-            logger.warn(msg);
+            logger.info(msg);

```

Comment:
I think we should go directly for debug level as the message is anyhow logged as a thing status update.